packages/components/src/errorSearchIllustration/ErrorSearchIllustration.native.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React from 'react';
+import { Image } from 'react-native';
+import { StyleSheet } from '@kiwicom/universal-components';
+
+// $FlowFixMe - Cannot resolve module RelativeImageStub ( possible solution https://bit.ly/2BYfHRr )
+import ErrorSearchTravel from './assets/mobile/error-search-travel.png';
```

Comment:
If you look here, it's also possible to use different pixel density pictures: https://github.com/kiwicom/mobile/tree/master/app/MMB/src/scenes/tripServices/insurance/paymentScene/images

Reply:
As I see there are already `@2x` and`@3x` versions inside assets subfolder, so I think it should be ok.